docs/topics/cache.txt

```diff
@@ -1178,45 +1178,34 @@ decorator)::
 
         return response
 
-There are a few other ways to control cache parameters. For example, HTTP
-allows applications to do the following:
-
-* Define the maximum time a page should be cached.
-
-* Specify whether a cache should always check for newer versions, only
-  delivering the cached content when there are no changes. (Some caches
-  might deliver cached content even if the server page changed, simply
-  because the cache copy isn't yet expired.)
-
-In Django, use the ``cache_control`` view decorator to specify these cache
-parameters. In this example, ``cache_control`` tells caches to revalidate the
-cache on every access and to store cached versions for, at most, 3,600 seconds::
+Generally, HTTP's caching mechanisms give you a lot of control over downstream
```

Comment:
@timgraham “Generally” is intended to highlight that the preceding `public`/`private` example is just an instance of a general Cache-Control mechanism. Can you suggest a better wording? I thought about “More generally, ...”.